src/libp2p_crypto.erl

```diff
@@ -374,6 +484,93 @@ base58check_decode(B58) ->
 
 -include_lib("eunit/include/eunit.hrl").
 
+multisig_test() ->
```

Comment:
This eunit seems like it _wants_ to be a common test imho...

Reply:
Yeah, probably...

Reply:
I no longer think so, now that I switched to generating test cases (mainly because the conglomerate one grew so large (because combos of M, N, hash and key types) that timeouts became an issue).